addons/sale/models/sale_order.py

```diff
@@ -884,10 +884,13 @@ def action_quotation_send(self):
         mail_template = self._find_mail_template()
```

Comment:
You can skip the call to `_find_mail_template` if the context key is provided ;)

Reply:
Okay :+1:

Reply:
Hello, @Feyensv I've checked the solution, and there's an issue with the `action_quotation_send` method. It's designed to handle a single record, but we need the action available in the list view as well. When attempting to perform the `Send an email` action with multiple records selected, it triggers a singleton error. How do you suggest i proceed here?

Reply:
Pushed a little fixup commit to handle the multi-records recorsets in `action_quotation_send` ;)